content/pt/docs/concepts/storage/persistent-volumes.md

```diff
@@ -0,0 +1,738 @@
+{{< /note >}}
+
+Um PV pode especificar uma [afinidade de nó](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volumenodeaffinity-v1-core) para definir restrições em relação ao limite de nós que podem acessar esse volume. Pods que utilizam um PV serão somente reservados para nós selecionados pela afinidade de nó.
+
+### Estado
+
+Um volume sempre estará em dos seguintes estados:
+
+* Available -- um recurso que está livre e ainda não foi atrelado a nenhuma requisição
+* Bound -- um volume atrelado a uma requisição
+* Released -- a requisição foi deletada, mas o curso ainda não foi recuperado pelo cluster 
+* Failed -- o volume fracassou na sua recuperação automática
+
+
+A CLI mostrará o nome do PV que foi atrelado à PVC
+
+## PersistentVolumeClaims
+
+Cada PVC contém uma `spec` e um status, que é a especificação e estado de uma requisição. O nome de um objeto PersistentVolumeClaim precisa ser um [DNS](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names) válido.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: myclaim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 8Gi
+  storageClassName: slow
+  selector:
+    matchLabels:
+      release: "stable"
+    matchExpressions:
+      - {key: environment, operator: In, values: [dev]}
+```
+
+### Modos de Acesso
+
+As requisições usam as mesmas convenções que os volumes quando eles solicitam um armazenamento com um modo de acesso específico.
+
+### Modos de Volume
+
+As requisições usam as mesmas convenções que os volumes quando eles indicam o tipo de volume, seja ele um sistema de arquivo ou dispositivo de bloco.
+
+### Recursos
+
+
+Assim como Pods, as requisições podem solicitar quantidades específicas de recurso. Neste caso, a solicitação é por armazenamento. O mesmo [modelo de recurso](https://git.k8s.io/community/contributors/design-proposals/scheduling/resources.md) vale para volumes e requisições.
+
+### Seletor
+
+Requisições podem especifiar um [seletor de rótulo](/docs/concepts/overview/working-with-objects/labels/#label-selectors) para posteriormente filtrar um grupo de volumes. Somente os volumes que possuam rótulos que satisfaçam os critérios do seletor podem ser atrelados à requisição. O seletor pode conter dois campos:
+
+* `matchLabels` - o volume deve ter um rótulo com esse valor
+* `matchExpressions` - uma lista de requisitos, como chave, lista de valores e operador relacionado aos valores e chaves. São operadores válidos: In, NotIn, Exists e DoesNotExist.
```

Comment:
To explain the meaning values, can use the glossary to: In, NotIn, Exists e DoesNotExist.